gobblin-aws/src/main/java/gobblin/aws/CloudInitScriptBuilder.java

```diff
@@ -0,0 +1,254 @@
+package gobblin.aws;
+
+import java.io.File;
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Optional;
+import com.google.common.base.Splitter;
+
+import gobblin.annotation.Alpha;
+import gobblin.cluster.GobblinClusterConfigurationKeys;
+import static gobblin.aws.GobblinAWSUtils.encodeBase64;
+
+
+/**
+ * Class to generate script for launching Gobblin cluster master and workers via cloud-init
+ * on EC2 instance boot up.
+ *
+ * @author Abhishek Tiwari
+ */
+@Alpha
+public class CloudInitScriptBuilder {
```

Comment:
Did you consider straight out writing a bash script with variables and then using this class to just set those variables?